src/ds/ds5/ds5-device.cpp

```diff
@@ -692,6 +695,27 @@ namespace librealsense
                 std::make_shared<thermal_compensation>(_thermal_monitor,thermal_compensation_toggle));
             }
 
+            auto const CUR_GVD_VERSION = gvd_buff[gvd_version_offset];
+            auto const MIN_GVD_VERSION = 0x4;
+
+            auto ir_filter_mask = d400_caps::CAP_IR_FILTER;
+            if (MIN_GVD_VERSION <= CUR_GVD_VERSION &&
+                val_in_range(_pid, {RS435_RGB_PID, RS435I_PID, RS455_PID}) &&
+                (_device_capabilities & ir_filter_mask) == ir_filter_mask)
+            {
+                device_name += "F";
```

Comment:
Please search a way to replace it with a regex substitute, maybe it will look better :)

Reply:
I tried to improve it with regex but did not find an optimal solution for it, in addition, we do not need flexibility there.
My suggestion is `device_name[device_name.size() - 1] = '6';`, what do you think about it?

Reply:
A regex use was added.
But I can find optimal logic for changing the name.
How much `include <regex>` increase our .cpp file?